src/index.js

```diff
@@ -1 +1,20 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getScores from './api/getScores.js';
+import postScores from './api/postScores.js';
+import showScores from './api/showScores.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/LPtxThYxLPjCftqWH3A6/scores';
+
+const refreshButton = document.querySelector('.btn-refresh');
+const form = document.getElementById('form-submission');
+
+form.addEventListener('click', async (ev) => {
+  ev.preventDefault();
+  postScores(url);
+});
```

Comment:
- [ ] You should add the event listener to submit button not the form itself. Currently, it starts making requests when I click on any form element. This is why you have duplicated items in the scores list.

Reply:
Hi @sinansevgi, 
Thank you for reviewing my pull request. I implemented what you asked in the comment. 

![code-d](https://user-images.githubusercontent.com/6615394/132775270-77eed757-c8bb-4ca7-830b-67672d3e7af9.PNG)

Reply:
This issue is well resolved! ✔️